services/basic_types.proto

```diff
@@ -1364,19 +1364,19 @@ message CurrentAndNextFeeSchedule {
  */
 message ServiceEndpoint {
     /**
-     * The 32-bit IPv4 address of the node encoded in left to right order (e.g.  127.0.0.1 has 127
-     * as its first byte)
+     * The 4-byte IPv4 address of the endpoint encoded in left to right order
+     * (e.g. 127.0.0.1 has bytes [127, 0, 0, 1])
      */
     bytes ipAddressV4 = 1;
 
     /**
-     * The port of the node
+     * The port of the service endpoint
      */
     int32 port = 2;
 
     /**
-     * A node domain name
-     * This MUST be the fully qualified domain name of the node.
+     * A node domain name.<br/>
```

Comment:
To add a line break after the sentence (otherwise markdown will merge the lines).
It could be a `<p>` tag (technically `<p/>` is more correct, but that confuses javadoc), and in future (much more thorough edits) that is under consideration for the standard separation between description and constraint.